docs/README.md

```diff
@@ -135,23 +136,23 @@ The table below contains the query string parameters that are available for you
 
 ### Parameters
 
-| Parameter | Description |
-| --- | --- |
-| `subscriptionId` | Subscription ID |
-| `subscriptionName` | Subscription name |
-| `offerId` | Offer ID |
-| `planId` | Plan ID |
-| `isFreeTrial` | Is this a free trial subscription? |
-| `seatQuantity` | The total number of seats available in this subscription |
-| `term_startDate` | The current subscription term start date |
-| `term_endDate` | The current subscription term end date |
-| `term_termUnit` | `P1M` for monthly term; `P1Y` for annual term |
-| `beneficiary_aadObjectId` | The Azure Active Directoy object ID (`sub` claim) of the user that the subscription was created for |
-| `beneficiary_aadTenantId` | The Azure Active Directory tenant ID (`tid` claim) of the user that the subscription was created for |
-| `beneficiary_userEmail` | The email address of the user that the subscription was created for |
+| Parameter | Description                                                                                                               |
```

Comment:
What are we doing here? Formatting?

Reply:
`Directoy` -> `Directoy` and formatting changed a little bit.